### PR TITLE
Add tech report for custom-instance-segmentation and update config

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**

### CODE REVIEW

**Comment by GalyaZalesskaya** [comment]
@snosov1 Hi, do you approve tech a report like that and sharing validation metrics of used datasets?
[Cityscapes](https://www.cityscapes-dataset.com/) - available for academic and personal research 
[Oxford Pets](https://www.robots.ox.ac.uk/~vgg/data/pets/) - available for research and commercial use 
[WGISD](https://github.com/thsant/wgisd#what-license-if-any-is-it-distributed-under) - non-commercial use, request to cite a paper in case of using this dataset

File: models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+| efficientnet_b2b-mask_rcnn-576x576 | 26.92 | 13.27 | 35.2 | 31.0 | 46.5 | [snapshot](https://storage.openvinotoolkit.org/repositories/openvino_training_extensions/models/instance_segmentation/v2/efficientnet_b2b-mask_rcnn-576x576.pth), [model template](efficientnet_b2b-mask_rcnn-576x576/template.yaml) | 1 |
+
+Average Precision (AP) is defined as an area under the precision/recall curve. 
+
+*NOTE* The Bbox AP and Segm AP metrics were calculated on the MS COCO dataset. The Average segm mAP was obtained as an average metric across 5 various datasets, which were chosen to test the generalization ability of provided training configs. To get metric information, which is achievable on each dataset, refer to this [spreadsheet](https://docs.google.com/spreadsheets/d/1wv3RzGu4Iwa0G_wCjBC5xMUMDr2JXEIm-2MeRwWCt7E/edit?usp=sharing). Datasets differ in size, the number of classes, train/val proportions, and the number of objects in the image.
```
Comment (snosov1) [inline models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md:16]: Can we have the spreadsheet in the doc itself? (instead of linking to google drive)

File: models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+| efficientnet_b2b-mask_rcnn-576x576 | 26.92 | 13.27 | 35.2 | 31.0 | 46.5 | [snapshot](https://storage.openvinotoolkit.org/repositories/openvino_training_extensions/models/instance_segmentation/v2/efficientnet_b2b-mask_rcnn-576x576.pth), [model template](efficientnet_b2b-mask_rcnn-576x576/template.yaml) | 1 |
+
+Average Precision (AP) is defined as an area under the precision/recall curve. 
+
+*NOTE* The Bbox AP and Segm AP metrics were calculated on the MS COCO dataset. The Average segm mAP was obtained as an average metric across 5 various datasets, which were chosen to test the generalization ability of provided training configs. To get metric information, which is achievable on each dataset, refer to this [spreadsheet](https://docs.google.com/spreadsheets/d/1wv3RzGu4Iwa0G_wCjBC5xMUMDr2JXEIm-2MeRwWCt7E/edit?usp=sharing). Datasets differ in size, the number of classes, train/val proportions, and the number of objects in the image. 
+
+The following source datasets were used in the experiments:
```
Comment (snosov1) [inline models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md:18]: You list 3 datasets here, but just above that you say the metric was calculated on 5 datasets.

File: models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
 
-| Model Name | Complexity (GFLOPs) | Size (Mp) | Bbox AP @ [IoU=0.50:0.95] | Segm AP @ [IoU=0.50:0.95] | Links | GPU_NUM |
-| --- | --- | --- | --- | --- | --- | --- |
-| efficientnet_b2b-mask_rcnn-480x480 | 14.8 | 10.27 | 34.1 | 29.4 | [snapshot](https://storage.openvinotoolkit.org/repositories/openvino_training_extensions/models/instance_segmentation/v2/efficientnet_b2b-mask_rcnn-480x480.pth), [model template](efficientnet_b2b-mask_rcnn-480x480/template.yaml) | 1 |
-| efficientnet_b2b-mask_rcnn-576x576 | 26.92 | 13.27 | 35.2 | 31.0 | [snapshot](https://storage.openvinotoolkit.org/repositories/openvino_training_extensions/models/instance_segmentation/v2/efficientnet_b2b-mask_rcnn-576x576.pth), [model template](efficientnet_b2b-mask_rcnn-576x576/template.yaml) | 1 |
+In the conducted experiments we got pre-trained on the MS COCO weights and then started the same fine-tuning on each dataset described above without freezing any layers.
```
Comment (snosov1) [inline models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md:23]: In the conducted experiments we used models pre-trained on the MS COCO and started the same fine-tuning on each dataset described above without freezing any layers.

File: models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md (hunk 1 above)
Comment (GalyaZalesskaya) [reply models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md:16]: Will do it

File: models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md (hunk 2 above)
Comment (GalyaZalesskaya) [reply models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md:18]: I write here (maybe not explicitly enough) that I use a Pets dataset to make two, just like with the WGISD dataset. That is why 3 original datasets were converted to 5 training

**Comment by Ilya-Krylov** [comment]
run tests

**Review by snosov1 — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
run tests
